ixmp/core.py

```diff
@@ -281,16 +295,18 @@ def timeslices(self):
                             columns=FIELDS['get_timeslices'])
 
     def add_timeslice(self, name, category, duration):
-        """Define a sub-annual time slice including a category and duration.
+        """Define a subannual timeslice including a category and duration.
+
+        See :method:`timeslices` for a detailed description of timeslices.
```

Comment:
What happens if the name is already defined?

As with units, suppose I accidentally define a timeslice with a typo. Can I delete it?

Reply:
> What happens if the name is already defined?

@zikolach, we could add something on the Python side like:
```
if name in self.timeslices.name:
    logger().info('Timeslice {} already defined, skipped this command'.format(name))
    return
... current function ...
```

> As with units, suppose I accidentally define a timeslice with a typo. Can I delete it?

No API yet, you'd have to do it directly via the underlying database. This is similar to the regions API. We'll add it at a later point when the usage becomes clearer. And before implementation of a delete feature, we'll need to discuss more: who can delete, how to treat existing data upon deletion, etc....